### ISSUE TITLE
Radio hangs on system start

### ISSUE THREAD

**Issue by aeickho** [travisgoodspeed/md380tools#96]
sometimes, after power on, the screen is white, no rx no tx is possible

**Comment by travisgoodspeed** [comment]
Have you found a way to reliably trigger this bug?

**Comment by aeickho** [comment]
I have the first time seen the error before about 4 weeks

**Comment by aeickho** [comment]
"1/20 times the error occurs" not now :(
30 restarts of 2 radios now "white-screen"

**Comment by travisgoodspeed** [comment]
Perhaps a recent patch is causing trouble, or stalling a thread?

**Comment by aeickho** [comment]
now I reboot the radio via usb ...

**Comment by aeickho** [comment]
Idea: I's  occurs only after firmware update

**Comment by aeickho** [comment]
@all can you report an "white screen" on an radio with stock firmware ?

**Comment by maushammer** [comment]
I've seen it twice out of maybe 70 power-ups with minimally-modded firmware (font changed only - I haven't applied any code patches)

Wild guess: does radio reception at time of powerup have anything to do with it?  Traffic is pretty light on my local repeater.

**Comment by M0IVQ** [comment]
I have seen this on stock firmware - if radio receives just after power on then locks up with white screen.

**Comment by aeickho** [comment]
If there are no more comments, I'll close this Issue in 3 days

**Comment by phl0** [comment]
I cannot confirm this issue when radio receives immediately after powering the radio. Switched it off and back on a few times while transmitting with a second radio and all working fine.

**Comment by oe3jtb** [comment]
Have it also somtimes, also with stock firmware, my explination was quick switchoff and switch  on

**Comment by travisgoodspeed** [comment]
I've seen it once or twice, but I haven't been able to reproduce it.  If we don't come up with a reproducible way by tomorrow, let's call it an act of god and close the issue.
